docs/Getting-Started.md

```diff
@@ -159,9 +159,6 @@ Unity brain name: 3DBallLearning
         Number of Visual Observations (per agent): 0
         Vector Observation space size (per agent): 8
         Number of stacked Vector Observation: 1
-        Vector Action space type: continuous
```

Comment:
Will there be no replacement here?

Reply:
Was waiting until we agreed on a rename. Will do as part of this PR.

Reply:
Oh wait, this is just the example log. It's all obsolete anyway.